### PR TITLE
Added -lh flag / LocalhostOnly property to just bind to the local loopback interface

### PR DESCRIPTION
**Pull request by tcnh**
Added -lh flag / LocalhostOnly property to just bind to the local loopback interface.
Effectively preventing wiki-access from other machines in the newtork.

Should address https://github.com/unclebob/fitnesse/issues/1141

Note: I am unable to test this change with SSL enabled

### CODE REVIEW

File: FitNesseRoot/FitNesse/UserGuide/AdministeringFitNesse/CommandLineArguments/content.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ A subset of configuration options can be provided through the command line. Comm
 | {{{-c <command>}}} | Command | ''none'' | Run a Rest Command and then exit. You can find more information on the -c options at RestfulServices. |
 | {{{-b <filename>}}} | !-RedirectOutput-! | ''stdout'' | redirect command output. This is most useful in conjunction with the ''-c'' option |
 | {{{-f <config file>}}} | n.a. | plugins.properties | Load an alternative configuration file. The file format adhere's to the java standard of property files. If this option is not provided, the default file ''plugins.properties'' will be loaded if it exists. |
-
+| {{{-lh}}} | !-LocalhostOnly-! | ''false'' | Bind only to the local loopback interface, preventing access to thewiki from other computers in the network |
```
Comment (fhoeben) [inline FitNesseRoot/FitNesse/UserGuide/AdministeringFitNesse/CommandLineArguments/content.txt]:
```suggestion
| {{{-lh}}} | !-LocalhostOnly-! | ''false'' | Bind only to the local loopback interface, preventing access to the wiki from other computers in the network |
```

**Comment by fhoeben** [comment]
Do you think you will be able to add a unit test showing a connection from localhost when the flag is set?

**Comment by tcnh** [comment]
Yes, I created one yesterday. I hope to find the time to polish and push it today

File: test/fitnesseMain/FitNesseMainTest.java [hunk 2]
```diff
@@ -112,13 +114,25 @@ public void runningCommandWithNonExistentAddressResultsInError() throws Exceptio
...
-        throw e;
+      Integer exitCode = new FitNesseMain().launchFitNesse(arguments);
+    } catch (Exception e) {
+      assertEquals("error loading page: 404", e.getMessage());
+      throw e;
     }
   }
 
+  @Test
+  public void localhostOnlyFlagResultsInConnectableFitnesseOnLocalHost() throws Exception {
```
Comment (fhoeben) [inline test/fitnesseMain/FitNesseMainTest.java]: Locally, a `./gradlew test` now fails on many tests. Does it work for you?

I believe this test keeps a server running. Is that indeed the case? If so, please ensure it is cleaned up after the test completes...

Comment (tcnh) [reply test/fitnesseMain/FitNesseMainTest.java]: I have no failing tests... May be OS dependent.
Let me see if I can force close the Fitnesse process

Comment (fhoeben) [reply test/fitnesseMain/FitNesseMainTest.java]: The `Shutdown` class will do it.
If I add a call to that (in a finally block) all tests pass...

Comment (fhoeben) [reply test/fitnesseMain/FitNesseMainTest.java]: Since I actually have the code available, let me just push it.

Comment (tcnh) [reply test/fitnesseMain/FitNesseMainTest.java]: Changed the test to call the shutdown responder. Could you verify if that prevents the other tests from failing?

Comment (fhoeben) [reply test/fitnesseMain/FitNesseMainTest.java]: You were faster than I expected. Testing now....

**Pull request merged**